features/ownership/get-creator.feature

```diff
@@ -0,0 +1,47 @@
+Feature: Test getting creator ownership
+    Background:
+        Given I am using the UDB3 base URL
+        And I am using an UiTID v1 API key of consumer "uitdatabank"
+        And I send and accept "application/json"
+
+    Scenario: Getting the creator of an ownership as creator
+        Given I am authorized as JWT provider v2 user "dev_e2e_test"
+        And I create a minimal organizer and save the "id" as "organizerId"
+        And I request ownership for "d759fd36-fb28-4fe3-8ec6-b4aaf990371d" on the organizer with organizerId "%{organizerId}" and save the "id" as "ownershipId"
+        When I send a GET request to "/ownerships/%{ownershipId}/creator"
+        Then the response status should be 200
+        And the JSON response at "userId" should be "auth0|64089494e980aedd96740212"
+        And the JSON response at "email" should be "dev+e2etest@publiq.be"
+
+    Scenario: Getting the creator of an ownership as a different user
+        Given I am authorized as JWT provider v2 user "dev_e2e_test"
+        And I create a minimal organizer and save the "id" as "organizerId"
+        And I wait for the organizer with url "/organizers/%{organizerId}" to be indexed
+        And I request ownership for "d759fd36-fb28-4fe3-8ec6-b4aaf990371d" on the organizer with organizerId "%{organizerId}" and save the "id" as "ownershipId"
+        And I approve the ownership with ownershipId "%{ownershipId}"
+
+        When I am authorized as JWT provider v2 user "invoerder"
+        And I send a GET request to "/ownerships/%{ownershipId}/creator"
+        Then the response status should be 200
+        And the JSON response at "userId" should be "auth0|64089494e980aedd96740212"
+        And the JSON response at "email" should be "dev+e2etest@publiq.be"
+
+    Scenario: Getting the creator of an ownership that you're not an owner of is not allowed
+        Given I am authorized as JWT provider v2 user "dev_e2e_test"
+        And I create a minimal organizer and save the "id" as "organizerId"
+        And I wait for the organizer with url "/organizers/%{organizerId}" to be indexed
+        # random user id
+        And I request ownership for "d920febc-8ff6-47af-a1f1-9897e166838f" on the organizer with organizerId "%{organizerId}" and save the "id" as "ownershipId"
```

Comment:
I think there is no need to create an ownership for a random user?

Reply:
As discussed through Slack it seems more logical to implement the endpoint `GET /organizers/{organizerId}/creator`. With this approach, it is not required to first create an ownership to get the creator of an organizer.

Reply:
[Refactor get creator handler](https://github.com/cultuurnet/udb3-backend/pull/1863/commits/dd39f63cefd46a667d2c92311d39393ca9d3a421)